docs/features/blocks/contentblocks/reusablecontent.rst

```diff
@@ -0,0 +1,24 @@
+Reusable Content Block
+======================
+
+Allows the use of reusable content snippets on any page 
+
+Field Reference
+---------------
+
+Fields and purposes:
+
+* **Content** - Chooser for reusable content
+
+.. figure:: img/reusable_content_block.png
+    :alt: The reusable content block
+
+    The reusuable content block in page editor
+
+Reusable Content is created in **Snippets > Reusable Content**. Once you have created a block of reusable content,
```

Comment:
This would be a good place to link to the reusable content snippets doc.

Reply:
done